src/servers/main.cc

```diff
@@ -282,6 +283,12 @@ std::vector<Option> options_{
      "Name of the model to be loaded on server startup. It may be specified "
      "multiple times to add multiple models. Note that this option will only "
      "take affect if --allow-model-control is true."},
+    {OPTION_TOTAL_PINNED_MEMORY_BYTE_SIZE, "total-pinned-memory-byte-size",
+     "The total byte size that can be allocated as pinned system memory. "
+     "If GPU support is enabled, the server will allocate pinned system memory "
+     "to accelerate data transfer between host and devices until it exceeds "
+     "the specified byte size. This option will not affect the allocation "
+     "conducted by the backend frameworks. Default is 0."},
```

Comment:
Default is tricky, but I think we should default to a non-zero value. Even on a system with not much CPU memory we should be able to have the default be 256MB.